src/ui/actions/auth/changePassword.js

```
@@ -0,0 +1,15 @@
+import { test } from '@playwright/test';
+import { SettingsPage } from '../../pages/auth/SettingsPage';
+import { HomePage } from '../../pages/HomePage';
+
+export async function changePassword(page, newPassword) {
+  await test.step(`Change user password`, async () => {
+    const settingsPage = new SettingsPage(page);
+    const homePage = new HomePage(page);
+
+    await settingsPage.open();
+    await settingsPage.submitChangePasswordForm(newPassword);
+
+    await homePage.assertYourFeedTabIsVisible();
```

After submitting the change password form you assert `Your Feed` visibility on the HomePage. For stronger verification of a successful password change consider also checking a success notification or that the settings page shows confirmation, or at least ensure navigation completed before asserting (e.g. wait for homePage to be visible). This reduces false-positives if the update action silently failed.